internal/replication/blockreplicator_3node_test.go

```diff
@@ -0,0 +1,313 @@
+package replication_test
```

Comment:
I see this pattern often too. Can't we use the same package name? Sometimes this leads to exporting methods unnecessarily.

Reply:
This is meant for black-box testing, which is what I do here on purpose. If I'll find I need intrusive tests that need access to non-exported functions, I'll add them in a separate test in the same package.